ToyBox/classes/MainUI/BagOfTricks.cs

```diff
@@ -179,6 +179,7 @@ public static class BagOfTricks {
                 () => UI.Toggle("Allow Item Use From Inventory During Combat", ref settings.toggleUseItemsDuringCombat),
                 () => UI.Toggle("Ignore Alignment Requirements for Abilities", ref settings.toggleIgnoreAbilityAlignmentRestriction),
                 () => UI.Toggle("Unlock the upper limit of 20 CL from class ", ref settings.toggleUnlockTheClFromClass, 0),
+                () => UI.Toggle("Remove Level 20 Caster Level Cap", ref settings.toggleUncappedCasterLevel),
```

Comment:
Does this conflict with the option on the previous line?  Let's have ArcaneTrixter and you reconcile these changes

Reply:
I rechecked the code and found that I added the same feature as Arcane
It's overlap again😫
I will try to delete the code of the overlapping part,